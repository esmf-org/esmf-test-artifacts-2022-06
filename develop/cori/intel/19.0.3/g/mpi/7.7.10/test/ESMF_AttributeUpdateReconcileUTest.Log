build time -- 2022-04-07 04:07:03
20220407 201204.830 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220407 201204.830 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220407 201204.830 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220407 201204.830 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220407 201204.830 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220407 201204.830 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220407 201204.830 INFO             PET0 Running with ESMF Version   : v8.3.0b12
20220407 201204.830 INFO             PET0 ESMF library build date/time: "Apr  7 2022" "04:05:27"
20220407 201204.830 INFO             PET0 ESMF library build location : /global/cscratch1/sd/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220407 201204.830 INFO             PET0 ESMF_COMM                   : mpi
20220407 201204.831 INFO             PET0 ESMF_MOAB                   : enabled
20220407 201204.831 INFO             PET0 ESMF_LAPACK                 : enabled
20220407 201204.831 INFO             PET0 ESMF_NETCDF                 : enabled
20220407 201204.831 INFO             PET0 ESMF_PNETCDF                : disabled
20220407 201204.831 INFO             PET0 ESMF_PIO                    : enabled
20220407 201204.831 INFO             PET0 ESMF_YAMLCPP                : enabled
20220407 201204.832 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220407 201204.832 INFO             PET0  NUMBER_OF_PROCESSORS           2
20220407 201204.834 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220407 201204.834 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220407 201204.834 INFO             PET0  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220407 201204.834 INFO             PET0 Finalizing ESMF
20220407 201204.830 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220407 201204.830 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220407 201204.830 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220407 201204.830 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220407 201204.830 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220407 201204.830 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220407 201204.830 INFO             PET1 Running with ESMF Version   : v8.3.0b12
20220407 201204.830 INFO             PET1 ESMF library build date/time: "Apr  7 2022" "04:05:27"
20220407 201204.830 INFO             PET1 ESMF library build location : /global/cscratch1/sd/rsdunlap/esmf-testing/intel_19.0.3_mpi_g_develop
20220407 201204.830 INFO             PET1 ESMF_COMM                   : mpi
20220407 201204.831 INFO             PET1 ESMF_MOAB                   : enabled
20220407 201204.831 INFO             PET1 ESMF_LAPACK                 : enabled
20220407 201204.831 INFO             PET1 ESMF_NETCDF                 : enabled
20220407 201204.831 INFO             PET1 ESMF_PNETCDF                : disabled
20220407 201204.831 INFO             PET1 ESMF_PIO                    : enabled
20220407 201204.831 INFO             PET1 ESMF_YAMLCPP                : enabled
20220407 201204.832 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220407 201204.832 INFO             PET1  NUMBER_OF_PROCESSORS           2
20220407 201204.834 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220407 201204.834 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220407 201204.834 INFO             PET1  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220407 201204.834 INFO             PET1 Finalizing ESMF
