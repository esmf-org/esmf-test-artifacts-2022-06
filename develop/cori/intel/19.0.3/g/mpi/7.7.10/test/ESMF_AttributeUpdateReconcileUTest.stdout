build time -- 2022-04-02 04:39:37
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  NUMBER_OF_PROCESSORS           2
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  NUMBER_OF_PROCESSORS           2
  PASS  PET            0 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            1 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            0 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  PASS  PET            1 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  PET           0  Test Elapsed Time    3.52500000000000       msec.
  PET           1  Test Elapsed Time    3.42500000000000       msec.
